components/Document/DocumentViewer.tsx

```diff
@@ -364,24 +424,62 @@ const DocumentViewer = ({
                 />
               )}
 
-              {pdfUrl ? (
+              {!isPost ? (
                 <>
-                  {!isPdfReady && <DocumentPlaceholder />}
-                  <PDFViewer
-                    pdfUrl={pdfUrl}
-                    scale={actualZoom}
-                    onReady={onPdfReady}
-                    contentRef={contentRef}
-                    viewerWidth={actualContentWidth}
-                    onLoadError={setHasLoadError}
-                    onPageRender={setLastPageRendered}
-                    numPagesToPreload={numPagesToPreload}
-                    showWhenLoading={
-                      <div style={{ padding: 20 }}>
-                        <DocumentPlaceholder />
-                      </div>
-                    }
-                  />
+                  {pdfUrl ? (
+                    <>
+                      {!isPdfReady && <DocumentPlaceholder />}
+
+                      <PDFViewer
+                        pdfUrl={pdfUrl}
+                        scale={actualZoom}
+                        onReady={onPdfReady}
+                        contentRef={contentRef}
+                        viewerWidth={actualContentWidth}
+                        onLoadError={setHasLoadError}
+                        onPageRender={setLastPageRendered}
+                        numPagesToPreload={numPagesToPreload}
+                        showWhenLoading={
+                          <div style={{ padding: 20 }}>
+                            <DocumentPlaceholder />
+                          </div>
+                        }
+                      />
+                    </>
+                  ) : (
+                    <div className={css(styles.uploadFile)}>
+                      <UploadFileDragAndDrop
+                        children={
+                          uploadingPdf ? (
+                            <FontAwesomeIcon
+                              icon={faSpinnerThird}
+                              spin
+                              style={{ fontSize: 30 }}
+                            />
+                          ) : (
+                            ""
+                          )
+                        }
+                        accept={".pdf"}
+                        beforeImageInstructions={
+                          <h3 className={css(styles.uploadFileTitle)}>
```

Comment:
I think we should tell them that we don't have the PDF since the user might expect us to have it. Maybe we put some caption saying "We couldn't automatically fetch the PDF"